source/nodejs/adaptivecards-designer/package.json

```diff
@@ -34,13 +34,13 @@
 		"release": "npm run build && webpack --mode=production && npm run dts"
 	},
 	"dependencies": {
+		"adaptivecards": "^2.10.0-beta.1",
```

Comment:
This was explicitly changed to a peer dependency instead of a regular dependency a while ago (I believe it was to let users upgrade ac version independently from designer) cc: @dclaux

Reply:
@marakim - can we change this to a peer dependency?

Reply:
Fixed